rules/performance/src/Rector/FuncCall/PreslashSimpleFunctionRector.php

```diff
@@ -75,6 +76,11 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        $autoImportNames = $this->parameterProvider->provideParameter(Option::AUTO_IMPORT_NAMES);
+        if ($autoImportNames) {
+            $this->parameterProvider->changeParameter(Option::AUTO_IMPORT_NAMES, false);
```

Comment:
I'm not sure if this approach should be taken here.
This disables an option that a user explicitly enabled, and would cause no class names to be imported.

Reply:
No, that will still works for import class, I created new proof fixture for it https://github.com/rectorphp/rector/pull/5268/commits/7f10e32e16ac969204643787df60108c3b7bbac2

Reply:
@samsonasik That fixture only tests importing class names, there is no function call, so the `PreslashSimpleFunctionRector` rector never runs. The issue comes in when the both are combined.

Can you please try the following test case (or something similar that uses a function call and a class import):

```php
<?php

namespace Rector\Performance\Tests\Rector\FuncCall\PreslashSimpleFunctionRector\Fixture;

class AutoImportNames
{
    public function run()
    {
        $count = count([]);
        $val = new \stdClass();
    }
}

?>
-----
<?php

namespace Rector\Performance\Tests\Rector\FuncCall\PreslashSimpleFunctionRector\Fixture;

use stdClass;
class AutoImportNames
{
    public function run()
    {
        $count = \count([]);
        $val = new stdClass();
    }
}

?>
```

Reply:
Thank you, I understood, I updated the fixture https://github.com/rectorphp/rector/pull/5268/commits/80e2025c9702f9192e9924c84fe3e349ef6ca4ca

Reply:
The possible safe solution I found right now is:

- keep use classname working
- skip add `\` prefix for function when auto import name enabled.

see https://github.com/rectorphp/rector/pull/5268/commits/8402fb7b3bb035cb7dd2bb7cd0513840297fe7aa